src/nxdoc/nuxeo-server/monitoring-and-maintenance/hotfixes-installation-notes-for-nuxeo-platform-lts-2016.md

```diff
@@ -133,12 +133,18 @@ If you have any questions, feel free to contact our support team via a dedicated
 
 ### Security Issue on MS Windows
 
-Windows users need to upgrade the Tomcat version to protect against a potential CSRF token leak in the default manager applications distributed with Tomcat. Follow this procedure to upgrade Tomcat to version `7.0.81`, version recommended by [NXP-19726](https://jira.nuxeo.com/browse/NXP-19726):
-1. Download the PowerShell script [upgrade_tomcat7.ps1](https://github.com/nuxeo/nuxeo/blob/8.10/nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/upgrade_tomcat7.ps1)
-2. In a PowerShell session, run the following command where NUXEO_HOME is the path to your Nuxeo instance
+Windows users need to upgrade the Tomcat version to protect against a potential CSRF token leak in the default manager applications distributed with Tomcat. Follow this procedure to upgrade Tomcat to version `7.0.81` (or above), version recommended by [NXP-19726](https://jira.nuxeo.com/browse/NXP-19726):
+1. Open a PowerShell session
+2. Ensure you can run "java" and "jar" with the current "$env:Path" environment variable or set the "$env:Path" as needed.
+3. Set the ExecutionPolicy to "Unrestricted" (see [MicroSoft usage notes](https://go.microsoft.com/fwlink/?LinkID=135170))
+4. run the script using the following command line format (using full path to NUXEO_HOME)
 ```
-upgrade_tomcat7.ps1 NUXEO_HOME 7.0.81
+client\scripts\upgrade_tomcat7.ps1 NUXEO_HOME 7.0.81
```

Comment:
why the **client\scripts\** path ?
the complete path would be %NUXEO_HOME%\templates\common-base\client\scripts